third_party/docfx-doclet-143274/src/test/resources/expected-generated-files/com.microsoft.samples.Link.yml

```diff
@@ -114,10 +114,13 @@ references:
   fullName: "com.microsoft.samples.Link.Link"
   package: "com.microsoft.samples"
 - uid: "java.lang.String"
+  href: "https://docs.oracle.com/javase/8/docs/api/java/lang/String.html"
```

Comment:
Should we be setting `isExternal: true`?

Reply:
Because I added the boolean `isExternal` field to the `SpecViewModel` it is default creating this field with false. Setting the `isExternal: true` will tell docfx to ignore the local references and use the xref maps instead. Since I am now providing the href in the local references, despite it technically be an external reference, I don't want it to look at the xref maps I want it to just use the local reference.